.circleci/config.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           paths: [ . ]
 
   test:
-    parallelism: 10
+    parallelism: 11
     machine:
```

Comment:
We are doing this because we re adding another container to run Oracle tests?

Reply:
Yep, exactly - that's right. That container by itself is already the slowest one - some of this is because the Oracle image is *massive* I think :(